app/assets/javascripts/discourse/templates/application.hbs

```diff
@@ -6,6 +6,7 @@
       {{custom-html "top"}}
     {{/if}}
     {{global-notice}}
```

Comment:
`global-notice` deals with static content that is displayed site wide for all users. 
`logs-notice` would need to keep track of a few properties to determine if it should be displayed. I don't think we should mix it into `global-notice`.